torntools/scripts/content/company/ttCompany.js

```diff
@@ -63,3 +65,13 @@ function showUserInfo() {
 			console.log("ERROR", err);
 		});
 }
+
+function changeEmployeesEffectivenessColor() {
+	requireElement("div.employee-list-wrap").then(() => {
+		for (let playerEffectiveness of doc.findAll("ul.employee-list div.effectiveness.clearfix")) {
+			if (playerEffectiveness.getAttribute("aria-label").split(":").map((x) => x.replace(/[a-zA-Z ;]/g, "")).map((x) => parseInt(x)).filter((x) => !isNaN(x)).some((x) => x < 0)) {
```

Comment:
This also triggers if you have a `-` in your name.

Adding `.slice(1)` after the `split` will throw away the name element from that array.

Reply:
Fixed.